BUILD.bazel

```diff
@@ -21,7 +21,7 @@ cc_library(
         "include/gnc/**/*.inl"
     ]),
     hdrs = glob([
-        "include/gnc/**/*.hpp", "include/orb/*.h", "include/orb/*.hpp"
+        "include/gnc/**/*.hpp", "include/orb/**/*.h", "include/orb/**/*.hpp"
```

Comment:
This changed allows Bazel to pick up the nested header files in the `orb` library.